{"issue_url": "https://api.github.com/repos/rails/rails/issues/9717", "repo": "rails", "body": "If I add `validates_uniqueness_of` then record will be invalid: `ActiveRecord::RecordInvalid: Validation failed: ... has already been taken`.\r\n\r\nNext steps, I gonna to request feature to take in account `mark_for_destruction` in validation uniqueness and others constraints.\r\n\r\nElsewhere the destroy before insert was in versions before 3.2.7. And I reverted it.", "_id": {"$oid": "523d1037bd354364b90045e5"}, "issue_id": 9717, "url": "https://api.github.com/repos/rails/rails/issues/comments/14898574", "html_url": "https://github.com/rails/rails/pull/9717#issuecomment-14898574", "updated_at": "2013-03-14T11:58:45Z", "user": {"subscriptions_url": "https://api.github.com/users/pftg/subscriptions", "events_url": "https://api.github.com/users/pftg/events{/privacy}", "repos_url": "https://api.github.com/users/pftg/repos", "gists_url": "https://api.github.com/users/pftg/gists{/gist_id}", "url": "https://api.github.com/users/pftg", "login": "pftg", "gravatar_id": "712a9cc852c438c0873ef0a98f749f2e", "html_url": "https://github.com/pftg", "following_url": "https://api.github.com/users/pftg/following{/other_user}", "received_events_url": "https://api.github.com/users/pftg/received_events", "organizations_url": "https://api.github.com/users/pftg/orgs", "avatar_url": "https://2.gravatar.com/avatar/712a9cc852c438c0873ef0a98f749f2e?d=https%3A%2F%2Fidenticons.github.com%2F58adcd3cfec4758f40b142a29c054d55.png", "starred_url": "https://api.github.com/users/pftg/starred{/owner}{/repo}", "id": 125715, "followers_url": "https://api.github.com/users/pftg/followers", "type": "User"}, "created_at": "2013-03-14T11:58:30Z", "id": 14898574, "owner": "rails"}
